frontend/package.json

```diff
@@ -117,7 +117,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint"
+      "pre-commit": "react-scripts test --watchAll=false"
```

Comment:
@GongDDEX 

- lint can't remove
- pre-commit should  support two steps, test and lint;